cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -905,6 +905,8 @@ spec:
             - name: ADVANCED_NETWORK_STATS
               value: "false"
             {{- end}}
+            - name: COLLECTIONS_MEMORY_CACHE_ENABLED
+              value: {{ (quote .Values.kubecostModel.collections.cache.enabled) | default (quote true) }}
```

Comment:
I don't see this present in the main values file. Did I miss it?

Reply:
We still need to have it accounted for in the values file. The values file should be the comprehensive dictionary of all settable values, even if it isn't technically required. Can you please address in a follow-up PR?